news/_posts/2022-01-13-new-theme-announce.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: We are working on a new front office theme
+subtitle: Here are some more details about it
+date: 2022-01-13
+authors: [NeOMakinG]
+icon: icon-laptop
+tags: [news, theme, 8.0]
+---
+
+# Work on the new theme has started
+
+For a lot of time, the community asked us to rework the classic theme because many things were missing inside it. We announce that we are working on a new front office theme, and you can find [here](https://github.com/PrestaShop/theme-refacto/compare/develop...mparvazi:auto-rtl).
```

Comment:
```suggestion
For a long time, many community members pushed for a rework of the Classic front-office theme. The maintainer team is happy to announce that a new front-office theme is being built, and you can find [here](https://github.com/PrestaShop/theme-refacto/compare/develop...mparvazi:auto-rtl).
```
There's no "us" 😉  just "maintainers".
Also I dont think we should spread the message "we did it because you asked for it" else we'll give the false message "we're here to do what you request from us" and this is not open source.

The link you added is a diff between develop and mparvazi branch, is this on purpose?

Reply:
Wut nop, I should change the link